pkg/storage/stores/tsdb/chunkwriter.go

```diff
@@ -45,6 +52,9 @@ func (w *ChunkWriter) Put(ctx context.Context, chunks []chunk.Chunk) error {
 		if err := w.PutOne(ctx, chunk.From, chunk.Through, chunk); err != nil {
 			return err
 		}
+		if err := w.backupChunkWriter.PutOne(ctx, chunk.From, chunk.Through, chunk); err != nil {
```

Comment:
not sure if I missing something here. Do we need to call `PutOne` on backupChunkWriter here given that it's already being called within `PutOne` method?

Reply:
Ahh, right. Not sure how I missed that 🤦